testutil/testing_app.go

```diff
@@ -22,6 +23,13 @@ import (
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	crisistypes "github.com/cosmos/cosmos-sdk/x/crisis/types"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	// helpers from cosmos-sdk is not helpful enough, need ibc-go helpers to backup
+	// (which is just the old version of "github.com/cosmos/cosmos-sdk/simapp/helpers")
+	// (maybe they think it is too helpful, so they modified it in the new version to be less helpful)
+	ibcgohelpers "github.com/cosmos/ibc-go/v5/testing/simapp/helpers"
```

Comment:
What is the actual thing that is missing? GenTx?

Reply:
Yes, `GenTx` is missing.
BTW in a following PR #99 this import is removed since we need additional functionality for setting `FeeGranter` in tx